src/main/kotlin/no/nav/familie/ef/sak/beregning/BeregningController.kt

```diff
@@ -74,7 +75,8 @@ class BeregningController(private val stegService: StegService,
     @GetMapping("/{behandlingId}")
     fun hentBeregnetBeløp(@PathVariable behandlingId: UUID): Ressurs<List<Beløpsperiode>> {
         tilgangService.validerTilgangTilBehandling(behandlingId, AuditLoggerEvent.UPDATE)
-        return Ressurs.success(tilkjentYtelseService.hentForBehandling(behandlingId).tilBeløpsperiode())
+        val startDatoForVedtak = vedtakService.hentVedtak(behandlingId).perioder?.perioder?.minByOrNull { it.datoFra }?.datoFra
```

Comment:
Hvis man har andeler fra 01.2021 - 12.2021
Og så opphører man fra 07.2021, så lagres ikke noen perioder på vedtaket, og denne returnerer null
Hva er då ønskelig å returnere til frontend? 

Andelene som finnes i databasen er fra 01.2021 - 06.2021

Reply:
Nei, denne skal brukes til å vise innvilgede andeler i vedtak- og beregningsbildet, samt innvilgede andeler i brevet. Så "opphørt vedtak" skal ikke be om et nytt beregnet beløp